src/pages/certificates/CertificateGenerate.tsx

```diff
@@ -55,95 +63,110 @@ const CertificateGenerate: FC = () => {
-                    className="u-certificate-button"
-                    onClick={() => downloadText("lxd-ui.crt", certs.crt)}
-                  >
-                    Download crt
-                  </Button>
-                )}
-              </Col>
-            </Row>
-          </li>
-          <li className="p-stepped-list__item">
-            <Row>
-              <Col size={8}>
-                <h3 className="p-stepped-list__title">Import</h3>
-                <div className="p-stepped-list__content">
-                  <p>
-                    Download <code>lxd-ui.pfx</code> and import it into your
-                    browser.
-                  </p>
-                  <BrowserImport />
-                </div>
-              </Col>
-              <Col size={4}>
-                {certs && (
+                </Col>
+                <Col size={3}>
                   <Button
-                    className="u-certificate-button"
-                    onClick={() => downloadBase64("lxd-ui.pfx", certs.pfx)}
+                    onClick={createCert}
+                    appearance="positive"
+                    disabled={isGenerating || certs !== null}
+                    hasIcon={isGenerating}
+                    aria-label={`${
+                      isGenerating ? "Generating" : "Generate"
+                    } certificate`}
                   >
-                    Download pfx
+                    {isGenerating && (
+                      <Icon
+                        className="is-light u-animation--spin"
+                        name="spinner"
+                      />
+                    )}
+                    <span>{isGenerating ? "Generating" : "Generate"}</span>
                   </Button>
-                )}
-              </Col>
-            </Row>
-          </li>
-          <li className="p-stepped-list__item">
-            <h3 className="p-stepped-list__title">
-              All done <Icon name="success" />
-            </h3>
-          </li>
-        </ol>
+                  {certs !== null && <Icon name="success" />}
+                </Col>
+              </Row>
+            </li>
+            <li className="p-stepped-list__item">
+              <Row>
+                <Col size={3}>
+                  <h3 className="p-stepped-list__title">Trust</h3>
+                </Col>
+                <Col size={6}>
+                  <div className="p-stepped-list__content">
+                    <p>
+                      Download <code>lxd-ui.crt</code> and add it to the LXD
+                      trust store
+                    </p>
+                    <div className="p-code-snippet">
+                      <pre className="p-code-snippet__block--icon">
+                        <code>lxc config trust add Downloads/lxd-ui.crt</code>{" "}
+                      </pre>
+                    </div>
+                  </div>
+                </Col>
+                <Col size={3}>
+                  {certs && (
```

Comment:
Maybe we could move the condition on the `Col`, and avoid rendering it empty when the certs haven't been generated?